package.json

```diff
@@ -114,7 +114,7 @@
         "classnames": "^2.2.6",
         "lodash": "^4.17.15",
         "moment": "^2.24.0",
-        "office-ui-fabric-react": "6.189.4",
+        "office-ui-fabric-react": "^6.199.0",
```

Comment:
It's intentional that the version number was originally a non-caret version string; it prevents folks doing `yarn add` for other packages from inadvertently updating `yarn.lock` with a mismatched version.

Reply:
Thanks Dan!

```suggestion
        "office-ui-fabric-react": "6.199.0",
```